README.md

```diff
@@ -1,17 +1,18 @@
 # Stargate
-An open source data gateway.
+
+#### An open source data gateway.
```

Comment:
```suggestion
#### An Open Source Data Gateway
```

I think, if we are going to make this a sub heading, we should remove the punctuation and title-case the text.